private/bufpkg/bufpluginexec/wasm_handler_test.go

```diff
@@ -0,0 +1,46 @@
+// Copyright 2020-2023 Buf Technologies, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package bufpluginexec
+
+import (
+	"errors"
+	"os"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestValidateWASMFilePath(t *testing.T) {
+	wasmPath := os.TempDir() + "/test.wasm"
+	notWasmPath := os.TempDir() + "/test.txt"
```

Comment:
nit: Might be nice to use t.TempDir() which will automatically get cleaned up.

Reply:
I'd say more than a nit heh :-)